spec/nameable/latin_spec.rb

```diff
@@ -1,284 +1,286 @@
+      end
+    end
 
-context('suffix patterns (formatted)') do
-  %w{Sr Sr. Senior}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Sr.')
+    %w{Jr Jr. Junior}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Jr.')
+      end
     end
-  end
 
-  %w{Jr Jr. Junior}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Jr.')
+    %w{Esq Esq. Esquire}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Esq.')
+      end
     end
-  end
 
-  %w{Esq Esq. Esquire}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Esq.')
+    %w{PHD PhD Ph.D Ph.D. P.H.D.}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Ph.D.')
+      end
     end
+
   end
 
-  %w{PHD PhD Ph.D Ph.D. P.H.D.}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq('Ph.D.')
+  context('suffix patterns (generational)') do
+    %w{ii III iv V VI ix Xiii}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq(suffix.upcase)
+      end
     end
   end
 
-end
-
-context('suffix patterns (generational)') do
-  %w{ii III iv V VI ix Xiii}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq(suffix.upcase)
+  context('suffix patterns (common)') do
+    %w{APR RPh MD MA DMD DDS PharmD EngD DPhil JD DD DO BA BS BSc BE BFA MA MS MSc MFA MLA MBA PE CSA CPA CPL CME CEng OFM CSV}.each do |suffix|
+      it suffix do
+        expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq(suffix.upcase)
+      end
     end
   end
-end
 
-context('suffix patterns (common)') do
-  %w{APR RPh MD MA DMD DDS PharmD EngD DPhil JD DD DO BA BS BSc BE BFA MA MS MSc MFA MLA MBA PE CSA CPA CPL CME CEng OFM CSV}.each do |suffix|
-    it suffix do
-      expect(Nameable::Latin.new.parse("Chris Horn #{suffix}").suffix).to eq(suffix.upcase)
+  context('suffix patterns (generic uppercase)') do
+    it 'generic DERP suffix' do
+      expect(Nameable::Latin.new.parse("Chris Horn DERP").suffix).to eq('DERP')
     end
-  end
-end
 
-context('suffix patterns (generic uppercase)') do
-  it 'generic DERP suffix' do
-    expect(Nameable::Latin.new.parse("Chris Horn DERP").suffix).to eq('DERP')
+    it 'no generic DERP suffix when all uppercase name' do
+      expect(Nameable::Latin.new.parse("CHRIS HORN DERP").suffix).to be_nil
+    end
   end
 
-  it 'no generic DERP suffix when all uppercase name' do
-    expect(Nameable::Latin.new.parse("CHRIS HORN DERP").suffix).to be_nil
+  context('multi word last names') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.